be/src/exprs/jit/jit_expr.cpp

```diff
@@ -75,6 +75,9 @@ Status JITExpr::prepare(RuntimeState* state, ExprContext* context) {
     }
     if (_jit_function != nullptr) {
         _jit_expr_name = _expr->debug_string();
+        if (_jit_expr_name.empty()) {
+            return Status::RuntimeError("[JIT] expr debug_string() is empty");
+        }
```

Comment:
it looks like a debug check, so using DCHECK here is better than returning an error?

Reply:
debug_string() is replaced by jit_func_name() in later PRs, `jit_func_name()` will process this exception.